releases.md

```diff
@@ -1,3 +1,59 @@
+## v2.6.0
+
+Date: 2021-12-29 <br>
+Contract: fio.reqobt <br>
+Hash: 10b5c26d6dec4da4cf43e5c966c65d437af3fe9fec76887fc3d4187c581e84b1 <br>
+CDT: fio.cdt (1.5.x) <br>
+Git Commit: fa07399925e322f1016a4f54c2937cb5cd593910
+
+Date: 2021-12-29 <br>
+Contract: fio.treasury  <br>
+Hash: 3b4ace746a3d9e6aa7f6f39ed29596956b111aa6dfe93db6b40bc1447c47230a <br>
+CDT: fio.cdt (1.5.x) <br>
+Git Commit: fa07399925e322f1016a4f54c2937cb5cd593910
+
+Date: 2021-12-29 <br>
+Contract: fio.staking  <br>
+Hash: 743eac5283c74c3dc30feaf27486104aabf182cd44437fbee827035a299e52bd <br>
```

Comment:
I get hash: efff312f5c6118fc1c5306848c8e8b32c240f0f7e0d176f03dd609d9c4bfb9cb

We updated the activation date, so this will be different from testnet.